Source/Reactions/ReactorArkode.cpp

```diff
@@ -199,31 +199,31 @@ ReactorArkode::react(
   if (use_erkstep == 0) {
```

Comment:
I think we can consolidate this now. Just the `arkode_mem = ` line in these 2 branches are different.

Reply:
These look different enough to me to just keep them. I think I could pull like 1 line out but then have to think of the BL_PROFILE stuff so I'm going to leave it.